src/Cli/dotnet/Commands/CliCommandStrings.resx

```diff
@@ -2476,4 +2479,20 @@ To display a value, specify the corresponding command-line option without provid
   <data name="SolutionAddReferencedProjectsOptionDescription" xml:space="preserve">
     <value>Recursively add projects' ReferencedProjects to solution</value>
   </data>
+  <data name="ToolExecuteCommandDescription" xml:space="preserve">
+    <value>Executes a tool from source without permanently installing it.</value>
+  </data>
+  <data name="ToolExecuteCommandMissingPackageId" xml:space="preserve">
+    <value>Missing package ID</value>
+  </data>
+  <data name="ToolDownloadConfirmationPrompt" xml:space="preserve">
+    <value>Tool package {0}@{1} will be downloaded from source {2}.
+Proceed? [y/n]</value>
```

Comment:
Perhaps we could reword this to be more localization friendly. Something like `Press 'y' to proceed or 'n' to decline.` This way, we can translate `proceed` and `decline`, and lock the y/n keys from being localized. I imagine for non-Latin keyboards, we may need to translate the Y/N as well to display.

Reply:
I think this is a good suggestion, but I personally prefer to be less verbose here.